tests/test_evidence_strings.py

```diff
@@ -244,6 +244,11 @@ def test_date(self):
         self.assertEqual(self.test_ges['evidence']['variant2disease']['date_asserted'], date_string)
         self.assertEqual(self.test_ges.date, date_string)
 
+    def test_validate(self):
+        test_args = get_args_CTTVGeneticsEvidenceString_init()
+        test_evidence_string = ES.CTTVGeneticsEvidenceString(*test_args)
+        test_evidence_string.validate()
```

Comment:
This checks if the EFO term is obsolete. "self.disease" returns an EFOTerm object (https://github.com/EBIvariation/eva-cttv-pipeline/blob/master/eva_cttv_pipeline/evidence_strings.py#L89):

>     @property
    def disease(self):
        return efo_term.EFOTerm(self['disease']['id'][0])

which has the function "is_obsolete" (https://github.com/EBIvariation/eva-cttv-pipeline/blob/master/eva_cttv_pipeline/efo_term.py#L75):

> def is_obsolete(self):
    if self.efoid is not None and self.efoid in EFOTerm.obsolete_terms:
        raise EFOTerm.IsObsoleteException("Term " + self.efoid + " is obsolete. Cause/Description/Action: " + EFOTerm.obsolete_terms[self.efoid])
        else:
            return False